### ISSUE TITLE
Don't move mouth

### ISSUE THREAD

**Issue by Novus1988** [Itokoyamato/TokoVOIP_TS3#277]
Hi I have a problem with using tokovoip. If I start the server everything works but the pg does not move his mouth while speaking, the only way I have to get it back to work is to restart the script while the server is already running then everything starts working again and the mouth does the animation. Do you have a solution for this? I use the latest version of both the plugin and the server side.

**Comment by valasszonnicknevet** [comment]
Same here. Subbed for solution.

**Comment by DanyCryEn** [comment]
same issue, you fixed this?

**Comment by valasszonnicknevet** [comment]
nope. looking for fix

**Comment by hickorysb** [comment]
I'll begin investigating in the morning.

**Comment by DanyCryEn** [comment]
> I'll begin investigating in the morning.

thanks

**Comment by hAKIZOR** [comment]
At game started if u do in console "restart tokovoip_script" it will work ;D so its just a problem of calling tokovoip resource in a bad position in your server.cfg

**Comment by DanyCryEn** [comment]
i've try on top of Server.cfg and on bottom......

**Comment by hAKIZOR** [comment]
> i've try on top of Server.cfg and on bottom......

yep im trying to solve :(.. for the moment just restart the resource xD  if u restart it works right?

**Comment by hickorysb** [comment]
Unable to replicate on 1.5.7, possibly fixed. I suggest waiting for that release. A lot of things are being fixed.

**Comment by hAKIZOR** [comment]
> Unable to replicate on 1.5.7, possibly fixed. I suggest waiting for that release. A lot of things are being fixed.
> 
> EDIT: Also unable to replicate on 1.5.4, possibly a client performance issue? More investigation needed.

i try your commit but is not fixed :(

**Comment by valasszonnicknevet** [comment]
Looks like the animation sections don't load when server starts. The ensure after server is online solve this issue for a few hours but not forever. 

Tried to start first and last but it didnt change the results.
Tried to start first and ensure in the end of .cfg but nothing happened. 

Only works if i restart / ensure it after ws server starts and write those infos into the console. (Doesn't count where u placed tokovoip in ur .cfg)

**Comment by hickorysb** [comment]
@valasszonnicknevet animations are loaded client side, so it's something client related. I just haven't figured out why it works on some clients and not others. The position of where it starts on the server shouldn't matter.

**Comment by hickorysb** [comment]
Do the radio animations work for you even if the mouth animations don't?

**Comment by hAKIZOR** [comment]
> Do the radio animations work for you even if the mouth animations don't?

Yes the radio animations work

**Comment by hickorysb** [comment]
> > Do the radio animations work for you even if the mouth animations don't?
> 
> Yes the radio animations work 

Possible that the animations are loading too soon for some clients.

**Comment by valasszonnicknevet** [comment]
The radio animations works for everybody. So i can see everyone talking on radio.
Only the mouth anims don't work.

**Comment by valasszonnicknevet** [comment]
Tried your commit. Problem is still avaiable for me too.

**Comment by hickorysb** [comment]
> Tried your commit. Problem is still avaiable for me too.

Alright, I think I know how to fix that.

**Comment by valasszonnicknevet** [comment]
How? :D

**Comment by hickorysb** [comment]
Check the most recent commit on that pull request and let me know if it is now working.

**Comment by hickorysb** [comment]
> Can't find that commit. 😞

https://github.com/Itokoyamato/TokoVOIP_TS3/pull/280/commits/be74e8f462270158a445e2fef3c6d430eb446853

**Comment by hAKIZOR** [comment]
> Check the most recent commit on that pull request and let me know if it is now working.

Not working :(

**Comment by hAKIZOR** [comment]
again work only if i restart the resource...

**Comment by valasszonnicknevet** [comment]
yap. animations don't work still.

**Comment by hickorysb** [comment]
Are you guys using extended mode?

**Comment by hickorysb** [comment]
Or essential mode?

**Comment by hAKIZOR** [comment]
> Are you guys using extended mode?

es extended v1 final

**Comment by hickorysb** [comment]
Ah, the script looks for a very specific convar value that esx/es overwrites. Could you go into your server and type "gametype" into the console without the quotes and then post the output here?

**Comment by hAKIZOR** [comment]
> Ah, the script looks for a very specific convar value that esx/es overwrites. Could you go into your server and type "gametype" into the console without the quotes and then post the output here?

"gametype" is "gta5"
default: ""
type: string

**Comment by hickorysb** [comment]
Alright it's not that then, I guess ESX stopped overwritting that.

**Comment by hickorysb** [comment]
I'm thinking at this point then that this is either related to a problem with specific clients, or an incompatibility with another resource. I'm going to pause on my investigation for now until 1.5.7 releases.

**Comment by valasszonnicknevet** [comment]
I tried your commit it didnt work.. i cleared my cache folder and now everyting work instantly after server starts.
So I think your commit works correctly just need to clear cache after the replace.

**Comment by grego-r-b** [comment]
Setting gametype to gta5 seems to have resolved animation issue for me
